backends/events.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function
```

Comment:
this is probably for compatibility with python2.7, I think you can use the regular print function.

Reply:
In fact let's just remove the import and the calls to `print` we won't need them.